app/classifier/tasks/index.spec.js

```diff
@@ -7,14 +7,14 @@ for (const key in tasks) {
   const TaskComponent = tasks[key];
   describe('Task ' + key, function() {
     let wrapper;
-  
+
     it('should render with default props', function() {
       wrapper = shallow(<TaskComponent />);
     });
-  
+
     it('should update on annotation change', function() {
       wrapper = shallow(<TaskComponent />);
-      let annotation = TaskComponent.getDefaultAnnotation()
+      let annotation = TaskComponent.getDefaultAnnotation(tasks[key])
```

Comment:
this now breaks when testing the combo, test fail message:
```
1) Task combo should update on annotation change:
TypeError: Cannot read property 'map' of undefined
at Function.getDefaultAnnotation (app/classifier/tasks/combo/index.cjsx:20:35)
at Context.<anonymous> (app/classifier/tasks/index.spec.js:37:38)
```

Reply:
I don't think `getDefaultAnnotation()` takes in a parameter, if it can be set up so it can, that would solve the issue.

Reply:
I think the combo task changed `getDefaultAnnotation` so that it could be invoked with a task, but I forget how that was implemented.

The error is something to do with an undefined array somewhere.

Reply:
This code change has changed a generic test for all tasks, so `tasks[key]` would have to be a valid task for each task type ie. it should be an array of tasks for the combo task.

Reply:
From this line, earlier in the test, `const TaskComponent = tasks[key];` `task[key` is a React component, and not a task description.

Reply:
After a bit of trial and error with the combo task, I think 7ef3266 fixes the tests by using the mock tasks from the dev classifier.